third_party/externals.bzl

```diff
@@ -186,9 +186,9 @@ def register_sorbet_dependencies():
     # In 2ddd7d791 (#7912) we upgraded the toolchain. Our old toolchain patches are on the `sorbet-old-toolchain` branch
     http_archive(
         name = "toolchains_llvm",
-        url = "https://github.com/sorbet/bazel-toolchain/archive/8d9165fd3560f6ff50bc4794972f714f4ba2adaa.tar.gz",
-        sha256 = "238b5a777bbfac3d5ec35cbd45ae2b84ca4118aa8f902ab27894912352e94658",
-        strip_prefix = "bazel-toolchain-8d9165fd3560f6ff50bc4794972f714f4ba2adaa",
+        url = "https://github.com/Shopify/bazel-toolchain/archive/5b4915c6730e86f21b1f2624f008a6bf9999910d.tar.gz",
+        sha256 = "103b72e9ece85c7d329143bc612804721c659e61808a412ceadcd96075cd1636",
+        strip_prefix = "bazel-toolchain-5b4915c6730e86f21b1f2624f008a6bf9999910d",
```

Comment:
Can we change this to a master commit, please?

Reply:
Rebased on top of the Sorbet branch: https://github.com/sorbet/bazel-toolchain/commits/sorbet/ 👍